docs/architecture/adr-061-liquid-staking.md

```diff
@@ -0,0 +1,88 @@
+
+The most important deficency of the legacy staking design is that it composes poorly with on chain protocols for trading, lending, derivatives that are referred to collectively as DeFi. The legacy staking implementation starves these applications of liquidity by increasing the risk free rate adaptively. It basically makes DeFi and staking security somewhat incompatible. 
+
+The Osmosis team has adopted the idea of Superfluid and Interfluid staking where assets that are participating in DeFi appliactions can also be used in proof of stake. This requires tight integration with an enshrined set of DeFi applications and thus is unsuitable for the Cosmos SDK.
+
+It's also important to note that Interchain Accounts are available in the default IBC implementation and can be used for staking and thus (rehypothecation)[https://www.investopedia.com/terms/h/hypothecation.asp#toc-what-is-rehypothecation] of delegations as a form of liquid staking is already possible. Thus liquid staking is already possible and these changes improve the UX of liquid staking. Centralized exahnges have also provided rehypthentication of staked assets and posed a challenge for decentralization. This ADR also take the position that liquid staking adoption is good and provides new levers to incentivize decentralization of stake. 
+
+These changes to the staking module have been in development for more than a year and have seen substantial industry adoption who plan to build staking UX. The internal economics at Informal team has also done a review of the impacts of these changes and this review led to the developement of the exempt delegation system. This system provides governance with a tunenable parameter for modulating the risks of pricipal agent problem called the exemption factor. 
+
+## Decision
+
+We implement the semi-fungible liquid staking system and exemption factor system within the cosmos sdk. These tokenizedshares have extremely limited fungibility to only the specific delegation record that was created when shares were tokenized. These assets can be used for OTC trades but composibiltiy with DeFi is limited. The primary expected use case is improving the user experience of liquid staking providers.
+
+A new governance parameter is introduced that defines the ratio of exempt to issued tokenized shares. This is called the exemption factor.
+
+Min self delegation is removed from the staking system with the expectation that it will be replaced by the exempt delegations sytem. The exempt delegation system allows multiple accounts to demonstrate economic alignment with the validator operator as team members, partners etc without co-mingling funds. The exempt delegation will likely be required to grow the validators business under widespread adoption of liquid staking and where governence sets the exemption factor.
```

Comment:
```suggestion
Min self delegation is removed from the staking system with the expectation that it will be replaced by the exempt delegations system. The exempt delegation system allows multiple accounts to demonstrate economic alignment with the validator operator as team members, partners etc. without co-mingling funds. Delegation exemption will likely be required to grow the validators' business under widespread adoption of liquid staking once governance has adjusted the exemption factor.
```